Assets/MixedRealityToolkit/Interfaces/InputSystem/IMixedRealityInputSystem.cs

```diff
@@ -306,60 +306,45 @@ public interface IMixedRealityInputSystem : IMixedRealityEventSystem
 
         #endregion Input Down
 
-        #region Input Pressed
-
-        /// <summary>
-        /// Raise Input Pressed.
-        /// </summary>
-        /// <param name="source"></param>
-        /// <param name="inputAction"></param>
-        void RaiseOnInputPressed(IMixedRealityInputSource source, MixedRealityInputAction inputAction);
-
-        /// <summary>
-        /// Raise Input Pressed.
-        /// </summary>
-        /// <param name="source"></param>
-        /// <param name="handedness"></param>
-        /// <param name="inputAction"></param>
-        void RaiseOnInputPressed(IMixedRealityInputSource source, Handedness handedness, MixedRealityInputAction inputAction);
+        #region Input Up
 
         /// <summary>
-        /// Raise Input Pressed.
+        /// Raise the input up event.
         /// </summary>
         /// <param name="source"></param>
         /// <param name="inputAction"></param>
-        /// <param name="pressAmount"></param>
-        void RaiseOnInputPressed(IMixedRealityInputSource source, MixedRealityInputAction inputAction, float pressAmount);
+        void RaiseOnInputUp(IMixedRealityInputSource source, MixedRealityInputAction inputAction);
 
         /// <summary>
-        /// Raise Input Pressed.
+        /// Raise the input up event.
         /// </summary>
         /// <param name="source"></param>
         /// <param name="handedness"></param>
         /// <param name="inputAction"></param>
-        /// <param name="pressAmount"></param>
-        void RaiseOnInputPressed(IMixedRealityInputSource source, Handedness handedness, MixedRealityInputAction inputAction, float pressAmount);
+        void RaiseOnInputUp(IMixedRealityInputSource source, Handedness handedness, MixedRealityInputAction inputAction);
 
-        #endregion Input Pressed
+        #endregion Input Up
 
-        #region Input Up
+        #region Float Input Changed
 
         /// <summary>
-        /// Raise the input up event.
+        /// Raise Float Input Changed.
         /// </summary>
         /// <param name="source"></param>
         /// <param name="inputAction"></param>
-        void RaiseOnInputUp(IMixedRealityInputSource source, MixedRealityInputAction inputAction);
+        /// <param name="inputValue"></param>
```

Comment:
><param name="inputValue"></param> [](start = 12, length = 33)

Can you leave some doccomments about the what this inputValue could be (i.e. is it between 0 and 1? Can it be negative? What does it represent?)

Or is it that it's so specific to the implementor that there's no reason to have this documented here?

Reply:
The architecture is so generic that it is very difficult to make any assertions on what this value may represent. I think all we can say is: _this is a float value reported by an input source_.

Reply:
Sounds good!